mempool/src/pool/try_get_fee.rs

```diff
@@ -1,80 +0,0 @@
-// Copyright (c) 2022 RBB S.r.l
-// opensource@mintlayer.org
-// SPDX-License-Identifier: MIT
-// Licensed under the MIT License;
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-use common::{chain::SignedTransaction, primitives::Idable};
-
-use crate::{error::TxValidationError, get_memory_usage::GetMemoryUsage};
-
-use super::{fee::Fee, Mempool};
-
-#[async_trait::async_trait]
-pub trait TryGetFee {
-    async fn try_get_fee(&self, tx: &SignedTransaction) -> Result<Fee, TxValidationError>;
-}
-
-#[async_trait::async_trait]
-impl<M> TryGetFee for Mempool<M>
-where
-    M: GetMemoryUsage + Send + Sync,
-{
-    // TODO this calculation is already done in ChainState, reuse it
-    async fn try_get_fee(&self, tx: &SignedTransaction) -> Result<Fee, TxValidationError> {
```

Comment:
nice! I've been recently in this function, but didn't notice that it is used only in tests and can be removed. I think it make sense now to also remove `ChainstateInterface::get_inputs_outpoints_coin_amount` and `ChainstateInterface::get_outputs_coin_amount` that were used here

Reply:
I removed `ChainstateInterface::get_outputs_coin_amount`, it's no longer used anywhere. However, `ChainstateInterface::get_inputs_outpoints_coin_amount` is still used in the test helper function implementation. Any more suggestions?

Reply:
no all good